app/script/strings.ts

```diff
@@ -523,6 +524,10 @@ export const errorHandlerStrings = defineMessages({
     defaultMessage: 'Something went wrong. Please contact your team administrator for details (Error 0).',
     id: 'BackendError.LABEL.SSO_GENERIC_ERROR',
   },
+  [LabeledError.GENERAL_ERRORS.LOW_DISK_SPACE]: {
+    defaultMessage: 'Not enough quota',
```

Comment:
Hmm, everywhere it says "Low disk space" but here you use the term "quota". I'd rather see a message like "Not enough disk space available" or something like this.

Reply:
Message is just temporary. Astrid and Roger will provide something nicer.